test.sh

```diff
@@ -1,10 +1,25 @@
+#!/bin/sh
```

Comment:
`sh` means different things on different systems.

It maps to:

* `ash` on Alpine Linux
* `dash` on Debian/Ubuntu
* `bash` on RHEL, Fedora, and Amazon Linux

If this should remain `sh`, then it should ensure broadly-compatible POSIX compliance. On the other hand, if we really mean `bash`, then we should just say `bash`.

Alpine doesn't ship with it, but it's a simple `apk add bash` from the package manager.

Reply:
Aren't all those shells Borne compatible?